lb_content_resolver/formats/dbcleanup.py

```diff
@@ -30,83 +30,83 @@ def databaseCleanup(file):
     cur = dbc.cursor()
     rows = []
 
-    print "Checking tracks"
+    print("Checking tracks")
     try:
         cur.execute("select track.id, location, release, artist from track, release_join where release_join.track = track.id order by track.id")
         rows = cur.fetchall()
-    except sqlite.OperationalError, msg:
-        print "Cannot get list of tracks: %s" % msg
+    except sqlite.OperationalError as err:
```

Comment:
The IDE moaned about this not being "python 3 like" and suggested this change so I went with it.